packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -130,6 +136,23 @@ export default class Sayt extends LitElement {
     }
   }
 
+  /**
+   * Dispatches an `sfx::autocomplete_fetch_data` event with the provided data.
+   * The event will only be dispatched if this SAYT component is the "correct" SAYT
+   * (see [[isCorrectSayt]]) and if the term is at least [[minSearchLength]] long.
+   *
+   * @param event An event that can contain a searchbox ID to identify its associated sayt.
```

Comment:
There's more than just the searchbox ID in this event, though. It also includes the search term. Perhaps something like this:

```suggestion
   * @param event The searchbox change event dispatched from the searchbox.
```